addons/steam_api/steam_i.gd

```diff
@@ -121,6 +127,12 @@ func clear_achievement(name:String) -> void:
 	user_stats.clear_achievement(name)
 	user_stats.store_stats()
 
+func get_num_achievement() -> int:
```

Comment:
Can you remove this. I only want to add really common used stuff to Steam namespace.